packages/react-server/src/ReactFlightStackConfigV8.js

```diff
@@ -9,23 +9,104 @@
 
 import type {ReactStackTrace} from 'shared/ReactTypes';
 
-import DefaultPrepareStackTrace from 'shared/DefaultPrepareStackTrace';
+let framesToSkip: number = 0;
+let collectedStackTrace: null | ReactStackTrace = null;
 
-function getStack(error: Error): string {
-  // We override Error.prepareStackTrace with our own version that normalizes
-  // the stack to V8 formatting even if the server uses other formatting.
-  // It also ensures that source maps are NOT applied to this since that can
-  // be slow we're better off doing that lazily from the client instead of
-  // eagerly on the server. If the stack has already been read, then we might
-  // not get a normalized stack and it might still have been source mapped.
-  const previousPrepare = Error.prepareStackTrace;
-  Error.prepareStackTrace = DefaultPrepareStackTrace;
-  try {
-    // eslint-disable-next-line react-internal/safe-string-coercion
-    return String(error.stack);
-  } finally {
-    Error.prepareStackTrace = previousPrepare;
+const identifierRegExp = /^[a-zA-Z_$][0-9a-zA-Z_$]*$/;
```

Comment:
Why do we need to test that? Asking because people already complain that our lint rules don't allow unicode characters in Component names so this may come up again here.

Reply:
I'm trying to mirror what V8 does as much as possible so we get the same from parsing and not.

https://github.com/v8/v8/blob/main/src/objects/call-site-info.cc#L767

However, I didn't dig into what isIdentifier does since you get lost in codegen. I just saw others do this. Usually what I do is this:

https://github.com/facebook/react/blob/main/packages/react-reconciler/src/ReactFiberHydrationDiffs.js#L212-L213